security/src/main/java/uk/ac/ox/ndph/mts/client/practitioner_service/PractitionerServiceClient.java

```diff
@@ -9,6 +9,6 @@
  */
 public interface PractitionerServiceClient {
 
-    List<RoleAssignmentDTO> getUserRoleAssignments(String userId);
+    List<RoleAssignmentDTO> getUserRoleAssignments(String userId, String token);
```

Comment:
Not blocking the approval on this PR, but can the token be injected by Spring? Maybe this could be something added to @tamirkamara client library. If not, this parameter ends up spreading all over the service-client codebase and ties the clients to the token method of authentication.